src/server/DeviceServer.js

```diff
@@ -87,9 +89,26 @@ class DeviceServer {
     this._eventPublisher = eventPublisher;
     this._areSystemFirmwareAutoupdatesEnabled =
       areSystemFirmwareAutoupdatesEnabled;
+
+    // Master listens to events from workers and broadcast them
+    // to all other workers except the owner.
+    if (useCluster && cluster.isMaster) {
+      (cluster: any).on('message', (eventOwnerWorker: Object, event: Event) => {
+        Object.values(cluster.workers).forEach((worker: any) => {
```

Comment:
Object.values() returns Array<mixed>, so `worker: mixed`,  I have to use any here or cast to any two times a few lines below.